lib/index.js

```diff
@@ -67,9 +68,24 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     optionallyAuthenticated,
     // FIXME add an implementation in bedrock-express
     boolParser(),
-    validate({query: 'account.getAccountsQuery'}),
+    intParser(),
+    validate({query: 'account.get'}),
     asyncHandler(async (req, res) => {
-      if(req.query.exists === true && req.query.email) {
+      /**
+       * @func getAccounts
+       * @param {Request} req
+       * @param {Response} res
+       * @param {Object} req.query
+       * @param {String} req.query.email - required to acces this route
+       * @param {Boolean} req.query.exists - used by Registration
+       * @param {Number} req.query.limit
+       * @param  {String} req.query.cursor - an account DID
```

Comment:
```suggestion
       * @param  {String} req.query.after - an account ID
```

Reply:
Also, reminder -- accounts don't use DIDs just regular IDs.

Reply:
are you sure? the account.id I see is: urn:uuid:473a4d07-2e91-420e-aed2-a12463e25679

which confirms to a did format.

Reply:
dids start with `did:`

Reply:
changing it.